### PR TITLE
War room Bug Fixes

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
War room Bug Fixes

### CODE REVIEW

File: modules/change_management/mmd.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "@context": "\/api\/3\/contexts\/StagingModelMetadata",
-    "@id": "\/api\/3\/staging_model_metadatas\/7f73f70b-50e2-402e-b612-a8cd4993c5d6",
```
Comment (cs-pooja-singh) [inline modules/change_management/mmd.json]: we can exclude this file

File: playbooks/02 - Use Case - CICD/Check Playbook is Active or Awaiting.json [hunk 2]
```diff
@@ -0,0 +1,161 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "Check Playbook is Active or Awaiting",
```
Comment (cs-pooja-singh) [inline playbooks/02 - Use Case - CICD/Check Playbook is Active or Awaiting.json]: Check if Playbook is Already Running

File: playbooks/02 - Use Case - CICD/Check Playbook is Active or Awaiting.json [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    "aliasName": null,
+    "tag": null,
+    "description": "Specify the playbook name and check whether the playbook is active or awaiting",
+    "isActive": true,
+    "debug": false,
+    "singleRecordExecution": false,
+    "remoteExecutableFlag": false,
+    "parameters": [
+        "playbook_name"
```
Comment (cs-pooja-singh) [inline playbooks/02 - Use Case - CICD/Check Playbook is Active or Awaiting.json]: can we base this on UUID instead?

File: playbooks/02 - Use Case - CICD/Create a Request (Issue).json [hunk 4]
```diff
@@ -219,20 +219,24 @@
...
+                                    "displayConditions": []
                                 },
                                 "searchable": false,
                                 "templateUrl": "app\/components\/form\/fields\/typeahead.html",
                                 "defaultValue": null,
                                 "_previousName": "",
+                                "inversedField": null,
                                 "playbookField": true,
-                                "displayTemplate": "{{ firstname }} {{ lastname }}",
+                                "displayTemplate": "%7B%7B%20firstname%20%7D%7D%20%7B%7B%20lastname%20%7D%7D",
```
Comment (cs-pooja-singh) [inline playbooks/02 - Use Case - CICD/Create a Request (Issue).json:237]: can we undo this?

File: playbooks/02 - Use Case - CICD/Create a Request (Issue).json [hunk 5]
```diff
@@ -258,7 +262,7 @@
                                 "defaultValue": null,
                                 "_previousName": "",
                                 "playbookField": true,
-                                "displayTemplate": "{{ firstname }} {{ lastname }}",
+                                "displayTemplate": "%7B%7B%20firstname%20%7D%7D%20%7B%7B%20lastname%20%7D%7D",
```
Comment (cs-pooja-singh) [inline playbooks/02 - Use Case - CICD/Create a Request (Issue).json:264]: undo this

File: playbooks/02 - Use Case - CICD/Create a Request (Issue).json [hunk 6]
```diff
@@ -219,20 +219,24 @@
...
                                 "searchable": false,
                                 "templateUrl": "app\/components\/form\/fields\/typeahead.html",
                                 "defaultValue": null,
                                 "_previousName": "",
+                                "inversedField": null,
                                 "playbookField": true,
-                                "displayTemplate": "{{ firstname }} {{ lastname }}",
+                                "displayTemplate": "%7B%7B%20firstname%20%7D%7D%20%7B%7B%20lastname%20%7D%7D",
                                 "lengthConstraint": false,
+                                "ownsRelationship": false,
```
Comment (cs-pooja-singh) [inline playbooks/02 - Use Case - CICD/Create a Request (Issue).json]: undo this

File: modules/change_management/mmd.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply modules/change_management/mmd.json]: done

File: playbooks/02 - Use Case - CICD/Check Playbook is Active or Awaiting.json (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/Check Playbook is Active or Awaiting.json]: done

File: playbooks/02 - Use Case - CICD/Create a Request (Issue).json (hunk 4 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/Create a Request (Issue).json:237]: No .. due to this we are not able to see the username in dropdown list

File: playbooks/02 - Use Case - CICD/Create a Request (Issue).json (hunk 5 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/Create a Request (Issue).json:264]: No .. due to this we are not able to see the username in dropdown list

File: playbooks/02 - Use Case - CICD/Create a Request (Issue).json (hunk 6 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/Create a Request (Issue).json]: done

**Pull request merged**